src/components/content/order/destroy/DestroyServiceStatusPolling.tsx

```diff
@@ -3,147 +3,128 @@
+import { useQuery } from '@tanstack/react-query';
+import { deploymentStatusPollingInterval } from '../../../utils/constants';
 
 function DestroyServiceStatusPolling({
     uuid,
+    isError,
     error,
-    isLoading,
+    isSuccess,
     setIsDestroying,
     setIsDestroyingCompleted,
 }: {
     uuid: string;
+    isError: boolean;
     error: Error | undefined;
-    isLoading: boolean;
+    isSuccess: boolean;
     setIsDestroying: (arg: boolean) => void;
     setIsDestroyingCompleted: (arg: boolean) => void;
 }): React.JSX.Element {
-    const getServiceDetailsByIdQuery = useServiceDetailsPollingQuery(uuid, [
-        ServiceDetailVo.serviceDeploymentState.DESTROY_FAILED,
-        ServiceDetailVo.serviceDeploymentState.DESTROY_SUCCESSFUL,
-    ]);
-
-    const stopWatch = useStopwatch({
-        autoStart: true,
+    const getServiceDetailsByIdQuery = useQuery({
+        queryKey: ['getServiceDetailsById', uuid],
+        queryFn: () => ServiceService.getServiceDetailsById(uuid),
+        refetchOnWindowFocus: false,
+        refetchInterval: uuid.length > 0 && isSuccess ? deploymentStatusPollingInterval : false,
+        enabled: uuid.length > 0 && isSuccess,
     });
 
     useEffect(() => {
-        if (
-            getServiceDetailsByIdQuery.data &&
-            [
-                ServiceDetailVo.serviceDeploymentState.DESTROY_FAILED,
-                ServiceDetailVo.serviceDeploymentState.DESTROY_SUCCESSFUL,
-            ].includes(getServiceDetailsByIdQuery.data.serviceDeploymentState)
-        ) {
+        if (isError) {
             setIsDestroying(false);
             setIsDestroyingCompleted(true);
         }
-    }, [getServiceDetailsByIdQuery.data, setIsDestroyingCompleted, setIsDestroying]);
+    }, [isError, setIsDestroyingCompleted, setIsDestroying]);
 
     useEffect(() => {
-        if (error) {
+        if (getServiceDetailsByIdQuery.isError) {
             setIsDestroying(false);
             setIsDestroyingCompleted(true);
         }
-    }, [error, setIsDestroyingCompleted, setIsDestroying]);
+    }, [getServiceDetailsByIdQuery.isError, setIsDestroyingCompleted, setIsDestroying]);
 
-    useEffect(() => {
-        if (getServiceDetailsByIdQuery.error) {
-            setIsDestroying(false);
-            setIsDestroyingCompleted(true);
+    if (isError) {
+        if (error instanceof ApiError && 'details' in error.body) {
+            const response: Response = error.body as Response;
+            return (
+                <div className={'submit-alert-tip'}>
+                    {' '}
+                    <Alert
+                        message={response.details}
+                        description={<OrderSubmitResultDetails msg={'Destroy request failed'} uuid={uuid} />}
+                        showIcon
+                        closable={true}
+                        type={'error'}
+                    />{' '}
+                </div>
+            );
         }
```

Comment:
if want to show the green alert for deployment progress, then pls also keep the timer. 
But the idea is to make it same as 'purge'. so no alert is needed. We just have to keep the 'loading' status of destroy button. 
In case of failure, show alert button.